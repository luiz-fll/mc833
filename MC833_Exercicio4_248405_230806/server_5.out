remoto: 127.0.0.1:6005
[SERVIDOR] Escutando em 127.0.0.1:6005
127.0.0.1:54426
GET / HTTP/1.0
Host: teste

127.0.0.1:54434
GET / HTTP/1.0
Host: teste

127.0.0.1:54436
GET / HTTP/1.0
Host: teste

127.0.0.1:54448
GET / HTTP/1.0
Host: teste

127.0.0.1:54456
GET / HTTP/1.0
Host: teste

127.0.0.1:54460
GET / HTTP/1.0
Host: teste

127.0.0.1:54470
GET / HTTP/1.0
Host: teste

127.0.0.1:54472
GET / HTTP/1.0
Host: teste

127.0.0.1:54478
GET / HTTP/1.0
Host: teste

