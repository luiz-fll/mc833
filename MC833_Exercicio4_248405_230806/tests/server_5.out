remoto: 127.0.0.1:50005
[SERVIDOR] Escutando em 127.0.0.1:50005
127.0.0.1:48810
GET / HTTP/1.0
Host: teste

127.0.0.1:48824
GET / HTTP/1.0
Host: teste

127.0.0.1:48836
GET / HTTP/1.0
Host: teste

127.0.0.1:48842
GET / HTTP/1.0
Host: teste

127.0.0.1:48854
GET / HTTP/1.0
Host: teste

127.0.0.1:48862
GET / HTTP/1.0
Host: teste

127.0.0.1:48874
GET / HTTP/1.0
Host: teste

127.0.0.1:48886
GET / HTTP/1.0
Host: teste

127.0.0.1:48902
GET / HTTP/1.0
Host: teste

127.0.0.1:48910
GET / HTTP/1.0
Host: teste

